src/proxy/reconnect.rs

```diff
@@ -169,20 +169,15 @@ where
     type Future = <Reconnect<M, T> as svc::Service<Req>>::Future;
 
     fn poll_ready(&mut self) -> Poll<(), Self::Error> {
-        match self.backoff {
-            Backoff::None => {}
-            Backoff::Fixed(_) => {
-                if let Some(delay) = self.active_backoff.as_mut() {
-                    match delay.poll() {
-                        Ok(Async::NotReady) => return Ok(Async::NotReady),
-                        Ok(Async::Ready(())) => {}
-                        Err(e) => {
-                            error!("timer failed; continuing without backoff: {}", e);
-                        }
-                    }
+        if let Some(delay) = self.active_backoff.as_mut() {
+            match delay.poll() {
+                Ok(Async::NotReady) => return Ok(Async::NotReady),
+                Ok(Async::Ready(())) => {}
+                Err(e) => {
+                    error!("timer failed; continuing without backoff: {}", e);
                 }
             }
-        };
```

Comment:
just to confirm... this was unrelated cleanup?

Reply:
Yea, just something I noticed when I was adding a debug log below...